lib/matplotlib/backends/backend_cairo.py

```diff
@@ -286,10 +286,18 @@ def get_canvas_width_height(self):
     def get_text_width_height_descent(self, s, prop, ismath):
         # docstring inherited
 
+        if ismath == 'TeX':
```

Comment:
I think just deferring to the super() implementation (as in my suggested patch) is enough?